src/components/Room/RoomContainer.tsx

```diff
@@ -26,11 +27,14 @@ function RoomContainer() {
 
   return (
     <div>
+      <Navbar/>
```

Comment:
i thought it made more sense to move Navbar here so that it can be styles separately from the 'contents' in the room. i moved it while i was working on the positioning of the main content / board / teams etc.